python/sqlflow_playground/playground_server_design.md

```diff
@@ -0,0 +1,136 @@
+# SQLFlow Playground Server
+
+SQLFlow Playground Server exposes a REST API service that enables users to share
+the resources in one playground cluster. Users can take advantage of SQLFlow by
+installing a small plugin on her/his Jupyter Notebook.
+
+This service is used to extend the SQLFlow Playground's capability, especially
```

Comment:
I can not understand why we need the `Playground server`.

> It clearly is not for
those who use the playground in single-user mode or those who just want to connect
to the SQLFlow server in the playground.

Do you mean the multi-user mode needs the playground server? Maybe  we should explain why can not 
use`jupyterhub`  the manage the resource on  k8s.

Reply:
No, multi-user mode does not need the playground server. I write this to warn that the service is only needed when we want to create a pod or do other cluster operations by ourselves (without jupyterhub).
This server supposes we do not have jupyterhub in the playground, and it totally is a backend service. I will make this more clear.